tests/e2e/asserts_test.go

```diff
@@ -2281,10 +2281,16 @@ func OfflineResizePVC(namespace, clusterName string, timeout int) {
 				// Deleting standby and replica pods
 				err = env.DeletePod(namespace, pod.Name, forceDelete)
 				Expect(err).ToNot(HaveOccurred())
-				// Ensuring cluster is healthy with three pods
-				AssertClusterIsReady(namespace, clusterName, timeout, env)
 			}
 		}
+		// Make sure the cluster phase changed first
+		AssertClusterEventuallyDepartsPhase(namespace, clusterName, apiv1.PhaseHealthy, timeout)
```

Comment:
are we sure this is always going to happen? I'm testing locally and the transition isn't there...

Reply:
Yeah, if we change grace_period=1, this test case failed everytime.